### PR TITLE
Compiled daemon, removed assets and torch dependency

### LINKED ISSUE #139: Compile Daemon
### Description

`pyinstaller` could be an option.

### PR DESCRIPTION
**Pull request by filopedraz**
### Description

- Closes #139 
- Removed `torch` dependency to check if GPU is available.
- Updated README.md with instructions to compile the daemon
- Removed assets folder. Moved static files to [static.premai.io repo](https://github.com/premAI-io/static.premai.io)

### Missing

- [x] GH action to generate the binaries cross-platform.
- [x] Test the daemon on GPU server

### CODE REVIEW

**Comment by filopedraz** [comment]
@casperdcl @tiero missing the GH actions. Can you help me out?

File: .github/workflows/on-release.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Release binaries
+on:
+  push: # TODO: remove?
```
Comment (casperdcl) [inline .github/workflows/on-release.yml:3]: unfortunately this workflow won't run before this PR is merged

**Review by casperdcl — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -80,3 +80,13 @@ You just need to run `premd` with `PREM_REGISTRY_URL` env variable.
 ```bash
 pytest
 ```
+
+## Compile the Daemon
+
+```bash
+virtualenv venv -p=3.10
```
[the remark this answers is not in the document]
Comment (filopedraz) [reply README.md:87]: As you prefer. No strong opinion on this.

File: .github/workflows/on-release.yml (hunk 1 above)
Comment (filopedraz) [reply .github/workflows/on-release.yml:3]: Let's merge it :)

**Comment by filopedraz** [comment]
Seems to work on GPU too now.

**Pull request merged**